### PR TITLE
HACKDAY-22 Jest tests

### PR DESCRIPTION
**Pull request by johnnolan**
Jest tests added to pipeline and able to run locally

### CODE REVIEW

File: .github/workflows/workflow_pr.yml [hunk 1]
```diff
@@ -25,6 +25,10 @@ defaults:
     shell: bash
 
 jobs:
+  test:
```
Comment (andrewpearce-digital) [inline .github/workflows/workflow_pr.yml:28]: lets add `test` to the `docker_build_scan_push` needs list on both workflows

File: .github/workflows/workflow_pr.yml [hunk 2]
```diff
@@ -25,6 +25,10 @@ defaults:
...
+    name: Jest Tests
+    uses: ./.github/workflows/test_job.yml
```
Comment (andrewpearce-digital) [inline .github/workflows/workflow_pr.yml:30]: I think this needs a `working_directory` setup here in both workflows too
```suggestion
    working_directory: ./service-add-lpa-to-pod
    uses: ./.github/workflows/test_job.yml
```

Comment (johnnolan) [reply .github/workflows/workflow_pr.yml:30]: Yep, just added and pushed that, will take a look at the docker change above

File: .github/workflows/workflow_pr.yml (hunk 1 above)
Comment (johnnolan) [reply .github/workflows/workflow_pr.yml:28]: Done 🤘

**Review by andrewpearce-digital — APPROVED** [review]

**Pull request merged**